### ISSUE TITLE
Server crash when Erebus antlion enters portal.

### ISSUE THREAD

**Issue by QQLazors** [Divine-Journey-2/main#399]
**Modpack version:**
2.5.0

**Java version:**
Java 8:
OpenJDK 1.8.0_292 for Ubuntu on the server.
Java 1.8.0_301-b09 for Windows on the client.

**Description:**
Erebus antlions cause a server crash when entering Erebus portals.

Later I tested spawning antlions and creepers and pushing them into the portal on seperate singleplayer worlds, and multiplayer servers. Antlions caused a crash, the creepers did not, although I am testing more extensively with other mobs from Erebus. I will update this with any further information.

I have received two distinct types of crash reports, examples of both types are linked to below.

**When did the crash happen?**
After exploring the Erebus dimension for some time, and attempting to return to the overworld, as well as any other attempts to move back and forth, until all hostile entities were killed.

Believed to be related to an antlion that wandered into the portal on the Erebus side, and then would travel through the portal in the opposite direction of the player when a player entered and loaded the area on the destination side of the portal.
The problem seemed to be resolved after "/cofh killall" was used in the server console to remove all hostile entities.

**Steps to reproduce:**

A video example of me reproducing the crash can be found here:
http://qqlazors.com/mods/crash-reports/DJ2_antlion_portal_crash.mp4

1. Join Server/Load World
2. Build Erebus portal.
3. Travel through at least once.
4. Spawn antlion using spawn egg.
5. Push into portal.
6. The server should crash.
7. On a multiplayer server, the server may be restarted and "/cofh killall" used in console to kill the antlion so that the affected player may rejoin without causing further crashes.

**Minecraft crash report:**

All encountered crash reports can be found here: 
http://qqlazors.com/mods/crash-reports/

They mainly fall into two types, the:
"Description: Exception ticking world entities
java.lang.NoClassDefFoundError: net/minecraft/entity/Entity$2"

And the:
"Description: Colliding entity with block
java.lang.StackOverflowError"

**Additional context:**
The server was recently updated to 2.5.0, using an old world file. 

Both the server and client have 32 GB of memory installed, and 8 GB allocated.

**Comment by Atricos** [comment]
This a very detailed reported, however I can't really do much about it. It seems to be a native Erebus bug, so report it to them please: https://github.com/vadis365/TheErebus/issues You could just copy and paste your entire message.

**Comment by QQLazors** [comment]
Yeah, I am going to put it over there once I finish testing things out. The thing is that there hasn't been any replies to any issues or updates in half a year, so honestly I'm not expecting a response in the near future. If I find a workaround or anything useful I'll leave it here.

**Comment by QQLazors** [comment]
As of current, I would suggest adding a quick message in the quest describing how to make the Erebus portal, perhaps telling the player to build around their portals to prevent errant mobs from wandering in.

This issue appears to extend to all ""Large"" mobs, including but not limited to the ones in the original post, but also ones like cyclops. I wouldn't want anybody to get stuck with this issue and not know how to prevent/fix it.
